security-questionnaire.md

```diff
@@ -2,35 +2,76 @@
 
 01.  What information might this feature expose to Web sites or other parties,
      and for what purposes is that exposure necessary?
+
+    - This feature allows web developers to establish WebTransport over HTTP/3
+      sessions to servers. For the purpose this feature exposes the network
+      circumstances and error information to some extent.
 02.  Do features in your specification expose the minimum amount of information
       necessary to enable their intended uses?
+
+    - Yes. Also, we're careful of not exposing information for endpoints that
+      don't support WebTransport over HTTP/3 as much as possible.
 03.  How do the features in your specification deal with personal information,
       personally-identifiable information (PII), or information derived from
       them?
+
+    - This feature itself doesn't deal with such sensitive information.
  04.  How do the features in your specification deal with sensitive information?
+
+    - This feature itself doesn't deal with such sensitive information.
  05.  Do the features in your specification introduce new state for an origin
       that persists across browsing sessions?
+
+    - No.
  06.  Do the features in your specification expose information about the
       underlying platform to origins?
+
+    - No.
  07.  Does this specification allow an origin to send data to the underlying
       platform?
+
+    - Yes, the feature uses network interfaces.
  08.  Do features in this specification enable access to device sensors?
+
+    - No.
  09.  What data do the features in this specification expose to an origin? Please
       also document what data is identical to data exposed by other features, in the
       same or different contexts.
+
+    - Arbitrary data sent from the server.
+    - Network errors (mostly opaque).
+    - Network information (indirectly).
  10.  Do features in this specification enable new script execution/loading
       mechanisms?
+
+    - No.
  11.  Do features in this specification allow an origin to access other devices?
+
+    - Yes.
  12.  Do features in this specification allow an origin some measure of control over
       a user agent's native UI?
+
+    - No.
  13.  What temporary identifiers do the features in this specification create or
       expose to the web?
+
+    - WebTransport
  14.  How does this specification distinguish between behavior in first-party and
       third-party contexts?
+
+    - The feature behaves in first-part and third-party contexts in the same way.
  15.  How do the features in this specification work in the context of a browser’s
       Private Browsing or Incognito mode?
+
+    - The feature works as usual.
```

Comment:
I think we should clarify that it is privacy-preserving by default. For example, "The feature works as usual, ie. no built-in authentication such as cookies is used."

We could say "Implementations do not share persistent connections or TLS session tickets between normal and Private Browsing or Incognito mode". It would be nice if this was specified somewhere.

Reply:
Done.